certif/app/components/certification-candidate-in-staging-item.hbs

```diff
@@ -2,51 +2,70 @@
   <td data-test-id='panel-candidate__lastName__add-staging'>
     <div class="certification-candidates-input-wrapper">
       <input
+        type="text"
+        required
         value="{{@candidateData.lastName}}"
-        class="certification-candidates-input"
+        class="certification-candidates-input {{if lastNameFocused 'focused'}}"
```

Comment:
avec le `this.` on indique d'où vient l'informations `lastNameFocused`, du coup je trouve ça plus explicite
```suggestion
        class="certification-candidates-input {{if this.lastNameFocused 'focused'}}"
```